test/yarn-auditer.js

```diff
@@ -27,6 +29,14 @@ function testDir(s) {
   return path.resolve(__dirname, s);
 }
 
+function canRunYarnBerry() {
+  const nodeVersion = childProcess
+    .execSync("node -v")
+    .toString()
+    .replace("\n", "");
+  return semver.gte(nodeVersion, "12.13.0");
+}
```

Comment:
Is there an advantage to calling this function for each of the three Yarn Berry tests rather than setting the `const` in the parent scope? I am not sure how expensive the Node `exec` is, but I think it's possible to avoid performing it repetitively.

Reply:
No, no advantage. I just thought it might be clearer to contain all that in a function rather than trying to put it in a one-liner or adding more file-level variables. But after updating it to a variable it looks alright.